src/lavatory/commands/stats.py

```diff
@@ -0,0 +1,20 @@
+"""Statistics of the repo."""
+import logging
+
+import click
+
+from ..utils.artifactory import Artifactory
+
+LOG = logging.getLogger(__name__)
+
+
+@click.command()
+@click.pass_context
+@click.option('--repo', default=None, required=True, help='Name of specific repository to run statistics against.')
```

Comment:
This should match the `--repo` in the `purge` command, able to accept multiple values or no value (to get all repos)